### ISSUE TITLE
New upgrade modals

### ISSUE THREAD

**Issue by ferristocrat** [storj/storj#5753]
## Motivation/Why?
- We want to be more clear what a user is getting when they "upgrade to pro"

## Acceptance Criteria

- When a user upgrades to a pro account they see the new modal.
  - If the user is on a mobile device they should see just the Pro Account details
  - If the user is on a wider screen they should see a comparison between the current (free) tier and Pro Account 
- User has the choice to use a credit card or deposit Storj tokens
- If a user deposits at least $50 USD value in Storj tokens, they should automatically be upgraded to a pro account

Note: The two upgrade paths from the Satellite UI are:
1. If the customer doesn't already have a payment method, when they click "+ Add New Payment Method" 
2. From the dashboard when the user clicks "Upgrade to pro"

## Figma Designs
- https://www.figma.com/file/HlmasFJNHxs2lzGerq3WYH/Satellite-GUI?node-id=22798%3A265513&t=PMqo5y611uDYeSrY-1

**Comment by ferristocrat** [comment]
> - If a user deposits at least $50 USD value in Storj tokens, they should automatically be upgraded to a pro account

Thinking we should pull this acceptance criteria out of this issue and create a separate issue/milestone for this flow

Thoughts @mobyvb @VitaliiShpital @boshevski ?

**Comment by mobyvb** [comment]
@ferristocrat I agree that upgrading accounts who pay in storj token to paid tier should be a separate task from this ticket (adding informational modal)

**Comment by ferristocrat** [comment]
> @ferristocrat I agree that upgrading accounts who pay in storj token to paid tier should be a separate task from this ticket (adding informational modal)

Added as out of scope
